longshot/application/main.tf

```diff
@@ -0,0 +1,217 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
+}
+
+variable "redis_type" {
+  default = "hobby-dev"
+}
+
+variable "database_name" {}
+
+variable "database_type" {
+  default = "db.t2.medium"
+}
+
+variable "database_scale" {
+  default = 100
+}
+
+variable "email_name" {}
+variable "email_address" {}
+
+variable "papertrail_destination" {}
+
+data "aws_ssm_parameter" "database_username" {
+  name = "/${var.name}/rds/username"
+}
+
+data "aws_ssm_parameter" "database_password" {
+  name = "/${var.name}/rds/password"
+}
+
+data "aws_ssm_parameter" "mandrill_api_key" {
+  name = "/mandrill/api-key"
+}
+
+resource "heroku_app" "app" {
+  name   = "dosomething-${var.name}"
+  region = "us"
+
+  organization {
+    name = "dosomething"
+  }
+
+  config_vars {
+    # App settings:
+    APP_ENV   = "production"
+    APP_DEBUG = "false"
+    APP_LOG   = "errorlog"
+    APP_URL   = "https://${var.host}"
+
+    # Drivers:
+    QUEUE_DRIVER   = "sqs"
+    CACHE_DRIVER   = "redis"
+    SESSION_DRIVER = "redis"
+    STORAGE_DRIVER = "s3"
+    MAIL_DRIVER    = "mandrill"
+
+    # Email:
+    EMAIL_NAME      = "${var.email_name}"
+    EMAIL_ADDRESS   = "${var.email_address}"
+    MAIL_HOST       = "smtp.mandrillapp.com"
+    MANDRILL_APIKEY = "${data.aws_ssm_parameter.mandrill_api_key.value}"
+
+    # Database:
+    DB_HOST     = "${aws_db_instance.database.address}"
```

Comment:
@DFurnes Checking that we don't have any special parameter groups for the RDS instance we need to specify here?

Reply:
Nope, this instance is just using the default parameter & option groups for MariaDB 10.0.

Reply:
Good to double-check! 👍